tests/unit/src/kakarot/precompiles/test_precompiles.cairo

```diff
@@ -35,3 +35,65 @@ func test__precompiles_should_throw_on_out_of_bounds{
 
     return ();
 }
+
+@external
+func test__is_precompile_should_return_false_when_address_is_greater_than_last_precompile{
```

Comment:
remove plz

Reply:
As soon as i come back to the hotel i would, sorry.